libs/domain/search/facet-navigation/facet-navigation.component.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  FacetComponentRegistryService,
+  FacetListService,
+} from '@spryker-oryx/search';
+import { FacetSelect } from '@spryker-oryx/search/facet';
+import { computed, hydratable, signal } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { tap } from 'rxjs/operators';
+import { SearchFacetNavigationOptions } from './facet-navigation.model';
+import { searchFacetNavigationStyles } from './facet-navigation.styles';
+
+@hydratable(['mouseover', 'focusin'])
+@defaultOptions({
+  expandedItemsCount: 5,
+  valueRenderLimit: 5,
+  minForSearch: 13,
+  bury: [{ facets: ['rating', 'price'] }],
+})
+export class SearchFacetNavigationComponent extends LayoutMixin(
+  ContentMixin<SearchFacetNavigationOptions>(LitElement)
+) {
+  static styles = [searchFacetNavigationStyles];
+
+  protected facetListService = resolve(FacetListService);
+  protected facetRenderer = resolve(FacetComponentRegistryService);
+  protected routerService = resolve(RouterService);
+
+  protected $facets = signal(this.facetListService.get());
+  protected filteredFacets = computed(() => {
+    const { bury } = this.$options();
+
+    return this.$facets()?.filter(
+      (facet) => !bury?.find((b) => b.facets.includes(facet.parameter))
+    );
+  });
+
+  protected override render(): TemplateResult | void {
+    const facets = this.filteredFacets();
+
+    if (!facets?.length) return;
+
+    const {
+      valueRenderLimit: renderLimit = Infinity,
+      expandedItemsCount = 0,
+      minForSearch = Infinity,
+    } = this.$options();
+
+    return html`${facets.map((facet, index) =>
+        this.facetRenderer.renderFacetComponent(
+          facet,
+          {
+            renderLimit,
+            open: index < expandedItemsCount,
+            minForSearch,
+            enableClear: !(
+              this.routerService.getPathId('category') &&
+              facet.parameter === 'category'
+            ),
+          },
+          this.applyFilters.bind(this)
+        )
+      )}
+      <style>
+        ${this.layoutStyles()}
+      </style> `;
+  }
+
+  protected applyFilters(e: CustomEvent<FacetSelect>): void {
+    const { name, value: selectedFacetValue } = e.detail;
+
+    const facet = this.$facets()?.find((facet) => facet.name === name);
+
+    if (!facet) return;
+    if (!selectedFacetValue) {
+      this.facetNavigation(facet.parameter);
+      return;
+    }
+
+    const values = facet.multiValued
+      ? [
+          ...(facet.selectedValues ?? []),
+          ...(selectedFacetValue.selected ? [selectedFacetValue.value] : []),
+        ].filter(
+          (selectedValue) =>
+            selectedFacetValue.selected ||
+            selectedValue !== selectedFacetValue.value
+        )
+      : [selectedFacetValue.value];
+
+    this.facetNavigation(facet.parameter, values as string[]);
+  }
+
+  protected facetNavigation(parameter: string, values: string[] = []): void {
```

Comment:
```suggestion
  protected navigate(parameter: string, values: string[] = []): void {
```

Don't think `facetNavigation` is a good method. `onSelect` could also work.

Reply:
`navigate` sounds good